maro/simulator/scenarios/supply_chain/units/storage.py

```diff
@@ -284,20 +284,23 @@ def try_add_products(
                     quantity = min(
                         int(quantity * fulfill_ratio_dict[storage_id]), self._remaining_space_dict[storage_id],
                     )
-                    self._add_product(sku_id, quantity)
-                    added_quantities[sku_id] = quantity
+                    if quantity > 0:
+                        self._add_product(sku_id, quantity)
+                        added_quantities[sku_id] = quantity
```

Comment:
How about move quantity check and `added_quantities[sku_id] = quantity` into `self._add_product`?

Reply:
not fix here